<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LeftOpenFileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFileButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRight">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textToEncrypt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>50</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter text to encrypt:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EncryptButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecryptButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEncryptionKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter key for encryption:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEnergy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter energy of encryption in range 0.05 to 1 (or click enter, default 0.5):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDecryptionKey">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter key for decryption:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RightOpenFileButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textKeyBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Key for decryption: </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textDecryptionMessageBrowser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>The decrypted message will be here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
